src/SplitIO/Component/Cache/Storage/Adapter/PRedis.php

```diff
@@ -24,13 +25,48 @@ public function __construct(array $options)
         }
 
         $_parameters = (isset($options['parameters'])) ? $options['parameters'] : null;
+        $_sentinels = (isset($options['sentinels'])) ? $options['sentinels'] : null;
         $_options = (isset($options['options'])) ? $options['options'] : null;
 
+        $_redisConfig = $this->getRedisConfiguration($_parameters, $_sentinels, $_options);
+
         if ($_options && isset($_options['prefix'])) {
             $_options['prefix'] = self::normalizePrefix($_options['prefix']);
         }
 
-        $this->client = new \Predis\Client($_parameters, $_options);
+        $this->client = new \Predis\Client($_redisConfig, $_options);
+    }
+
+    private function isValidSentinel($sentinels, $options)
```

Comment:
can we change to `isValidSentinelConfig` ?

Reply:
got it